pokemongo_bot/cell_workers/inventory_actions_worker.py

```diff
@@ -0,0 +1,47 @@
+import json
+import datetime
+import os.path
+
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot import logger
+
+
+class InventoryActionsWorker(object):
+
+    def __init__(self, bot):
+        self.config = bot.config
+        self.pokemon_list = bot.pokemon_list
+        self.api = bot.api
+        self.inventory_options_file = 'data/inventory_options'
+
+    def work(self):
+        if self.config.use_lucky_egg and (self.config.mode == "all" or
+                                          self.config.mode == "poke"):
```

Comment:
this suggests that someone doing evolution of a lot of pokemons for XP in "farm" mode and trying to use a lucky egg from your code will be denied. Changing the logic or explaining the feature on the readme seems important